packages/application-extension/schema/context-menu-suppressor.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "Context Menu Suppression",
+  "description": "Context Menu Suppression Settings.",
+  "jupyter.lab.menus": {},
+  "jupyter.lab.shortcuts": [],
+  "properties": {
+    "disabled": {
+      "title": "Whether the context menu is suppressed",
```

I guess we should clarify if we suppress browser or application (jupyterlab/notebook) menu?

```suggestion
      "title": "Whether the application context menu is suppressed",
```